client-app/src/admin/wip/treeMap/GridTreeMapModel.js

```diff
@@ -0,0 +1,96 @@
+import {XH, HoistModel, managed} from '@xh/hoist/core';
+import {LoadSupport} from '@xh/hoist/core/mixins';
+import {GridModel, emptyFlexCol} from '@xh/hoist/cmp/grid';
+import {DimensionChooserModel} from '@xh/hoist/desktop/cmp/dimensionchooser';
+import {TreeMapModel} from '@xh/hoist/desktop/cmp/treemap';
+import {numberRenderer, millionsRenderer, fmtNumberTooltip} from '@xh/hoist/format';
+
+@HoistModel
+@LoadSupport
+export class GridTreeMapModel {
+
+    @managed
+    dimChooserModel = new DimensionChooserModel({
+        dimensions: [
+            {value: 'region', label: 'Region'},
+            {value: 'sector', label: 'Sector'},
+            {value: 'symbol', label: 'Symbol'}
+        ],
+        initialValue: ['sector', 'symbol']
+    });
+
+    @managed
+    gridModel = new GridModel({
+        treeMode: true,
+        sortBy: 'pnl|desc|abs',
+        emptyText: 'No records found...',
+        enableColChooser: true,
+        compact: XH.appModel.useCompactGrids,
+        selModel: 'multiple',
+        columns: [
+            {
+                headerName: 'Name',
+                width: 200,
+                field: 'name',
+                isTreeColumn: true
+            },
+            {
+                field: 'mktVal',
+                headerName: 'Mkt Value (m)',
+                headerTooltip: 'Market value (in millions USD)',
+                align: 'right',
+                width: 130,
+                absSort: true,
+                agOptions: {
+                    aggFunc: 'sum'
```

Comment:
What's going on with these? Doesn't portfolioService produce with aggregates?

Also - with this grid and the other splitMap example - I vote to simplify the grid config itself as much as possible since it's not the code we're looking to highlight.  E.g. could drop the colChooser, tooltips, anything else not really essential.  (But still would keep e.g. the number formatting or anything where leaving it out will just look weird or distracting.)